test/test_linalg.py

```diff
@@ -1868,6 +1869,266 @@ def test_nuclear_norm_exceptions_old(self, device):
         self.assertRaisesRegex(RuntimeError, "duplicate or invalid", torch.norm, x, "nuc", (0, 0))
         self.assertRaisesRegex(IndexError, "Dimension out of range", torch.norm, x, "nuc", (0, 2))
 
+    # ~~~ tests for torch.svd ~~~
+    @skipCUDAIfNoMagma
+    @skipCPUIfNoLapack
+    @dtypes(torch.double)
+    def test_svd(self, device, dtype):
+        def run_test(dims, some, compute_uv):
+            x = torch.randn(*dims, dtype=dtype, device=device)
+            outu = torch.tensor((), dtype=dtype, device=device)
```

Comment:
`torch.tensor((), ...` -> `torch.empty(0, ...)`

Reply:
fixed by 3ddb6accfc.
But I'm curious: why is `empty` better than `tensor`?

Reply:
In this case they're equivalent, it's just more consistent to use empty when the values aren't relevant.